powerschool-plugin/description.txt

```diff
@@ -0,0 +1,8 @@
+Plugin management is done through Powersource, at https://support.powerschool.com/exchange/myFiles.action - the plugin.xml file in this directory should be uploaded as the file, and the data below filled in for the other details.
```

Comment:
Is this description file also part of the plugin (i.e. used by Powerschool), or is it just a readme for our purposes?

Reply:
This is only for our purposes, and I'll update this file with the rest of the plugin config info.